streams/shared/src/main/scala/zio/stream/Sink.scala

```diff
@@ -242,13 +242,9 @@ object Sink {
   /**
    * see [[ZSink.succeed]]
    */
-  final def succeed[B](b: B): Sink[Nothing, Nothing, Any, B] =
+  final def succeed[A, B](b: B): Sink[Nothing, A, A, B] =
```

Comment:
`A` should be reverted now. 😄

Reply:
It's actually required because of the leftovers.

Reply:
Sorry yes, I skimmed over the implementation.